CONTRIBUTE.md

```diff
@@ -7,49 +7,87 @@
 ### Contribution of OEO content
 Please read [OEO wiki](https://github.com/OpenEnergyPlatform/ontology/wiki/Best-Practice-Principles) carefully.
 
-### Philosophy
+### Workflow
 
 The development of a feature for this repository is inspired from the workflow described 
 by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
 
-1. Create [an issue](https://help.github.com/en/articles/creating-an-issue) on the github repository to describe the addition to the ontology. Discussion about the implementation details should occur within this issue 
+1. Create [an issue](https://help.github.com/en/articles/creating-an-issue) on the github repository
```

Comment:
Would this mean that I could create an issue, and start working on the ontology in my branch right away? Or would I need to wait until someone has commented on my issue? Maybe this is something we should write up?

Reply:
One can (kann) always work on the branch, there is no way to forbid that. What we could say in the workflow is that someone cannot (darf nicht) start working on the branch before they discussed with someone else within the issue

Reply:
We should maybe only enforce this for issues of type A or B

Reply:
Done in 8d8c4f76cff00a80c523ca719fa6c7f29edbaca3